.github/workflows/crypto-tests.yml

```
@@ -0,0 +1,33 @@
+    branches: [ main, develop ]
+    paths:
+      - 'src/lib/zkp/**'
+      - 'circuits/**'
+      - 'tests/crypto/**'
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)